src/core/Form/Toggle/ToggleButton/ToggleButton.md

```diff
@@ -1,34 +1,128 @@
+`<ToggleButton>` is a toggle switch that is semantically a button. It can be used to switch between two mutually exclusive states or actions.
```

Comment:
```suggestion
`<ToggleButton>` is a toggle switch that is semantically a button. It can be used to switch between two mutually exclusive states or actions. The state of the button is communicated to assistive technology using `aria-pressed`.
```